### ISSUE TITLE
Kong Getting started tutorial not working on EKS

### ISSUE THREAD

**Issue by sjtu-biggie** [Kong/kong#10375]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Kong version (`$ kong version`)

Latest

### Current Behavior

I follow the getting started tutorial https://docs.konghq.com/kubernetes-ingress-controller/2.8.x/guides/getting-started/ 
to deploy an HTTP application and configure a route to it. 
Making request to Kong Gateway with no url parameter works fine
```
curl -i $PROXY_IP
HTTP/1.1 404 Not Found
Content-Type: application/json; charset=utf-8
Connection: keep-alive
Content-Length: 48
X-Kong-Response-Latency: 0
Server: kong/3.0.0
{"message":"no Route matched with those values"}
```
However, when I run:
```
curl -i http://kong.example/echo --resolve kong.example:80:$PROXY_IP
curl: (6) Could not resolve host: kong.example
```
Then I found that the $PROXY_IP derived according to the tutorial in EKS is a hostname
```
$ echo $PROXY_IP
<secret>.elb.us-east-1.amazonaws.com
```
and `curl --resolve` can not take hostname as the target ip. 

To finish the tutorial in this scenario, we need to get the real AWS load balancer IP of that hostname
```
$ getent hosts $PROXY_IP
xxx.xxx.xxx.xxx   <secret>.elb.us-east-1.amazonaws.com
xxx.xxx.xxx.xxx <secret>.elb.us-east-1.amazonaws.com
xxx.xxx.xxx.xxx   <secret>.elb.us-east-1.amazonaws.com
```
This time it works fine.

To sum up, the `$PROXY_IP` derived from the tutorial https://docs.konghq.com/kubernetes-ingress-controller/2.8.x/deployment/eks/ is a hostname, which will cause resolution failure in the later stage https://docs.konghq.com/kubernetes-ingress-controller/2.8.x/guides/getting-started/.
Proper hint or modification should be added. This issue may happen in other public cloud kubernetes solutions.

**Comment by fffonion** [comment]
@sjtu-biggie This does look like an issue for docs. PRs are welcomed to submit to https://github.com/kong/docs.konghq.com!
